util.ts

```diff
@@ -1,16 +1,42 @@
+import moment from "moment";
```

Comment:
Moment needs to be imported from `obsidian` instead so we don't bundle a copy into the plugin.